RNSnackbar.podspec

```diff
@@ -0,0 +1,20 @@
+require 'json'
+
+package = JSON.parse(File.read('package.json'))
+
+Pod::Spec.new do |s|
+  s.name                = 'RNSnackbar'
+  s.version             = package['version']
+  s.summary             = package['description']
+  s.description         = package['description']
+  s.homepage            = package['homepage']
+  s.license             = package['license']
+  s.author              = package['author']
+  s.source              = { :git => 'https://github.com/cooperka/react-native-snackbar.git' }
```

Comment:
Can we also include a tag (e.g. `:tag => "#{s.version}"`)? I assume it would pull straight from master unless we do this.